project/AppDependencies.scala

```diff
@@ -17,20 +17,22 @@
 import sbt._
 
 object AppDependencies {
-  def apply(): Seq[ModuleID] = CompileDependencies() ++ UnitTestDependencies() ++ IntegrationTestDependencies()
+  def apply(): Seq[ModuleID] = CompileDependencies() ++ UnitTestDependencies()
+
+  val it: Seq[ModuleID] = Seq()
+
 }
 
 object CompileDependencies {
-  val domainVersion = "8.1.0-play-28"
-  val bootstrapVersion = "7.12.0"
-  val hmrcMongoVersion = "0.74.0"
+  val domainVersion = "8.3.0-play-28"
```

Comment:
@Michael-RJ Please use 30 instead of 28,
libraryDependencies += "uk.gov.hmrc" %% "domain-play-30" % "9.0.0"